app/src/sharedTest/java/org/oppia/app/topic/info/TopicInfoFragmentTest.kt

```diff
@@ -67,17 +69,29 @@ class TopicInfoFragmentTest {
   }
 
   @Test
-  @Ignore("Landscape not properly supported") // TODO(#56): Reenable once landscape is supported.
   fun testTopicInfoFragment_loadFragment_configurationChange_checkTopicName_isCorrect() {
-    launchTopicActivityIntent(TEST_TOPIC_ID).use { scenario ->
-      scenario.onActivity {  activity ->
-        activity.requestedOrientation = ActivityInfo.SCREEN_ORIENTATION_LANDSCAPE
-      }
-
+    launchTopicActivityIntent(TEST_TOPIC_ID).use {
+      onView(ViewMatchers.isRoot()).perform(orientationLandscape())
       onView(withId(R.id.topic_name_text_view)).check(matches(withText(containsString(TOPIC_NAME))))
     }
   }
 
+  @Test
+  fun testTopicInfoFragment_loadFragment_configurationLandscape_isCorrect() {
+    launchTopicActivityIntent(TEST_TOPIC_ID).use {
+      onView(ViewMatchers.isRoot()).perform(orientationLandscape())
+      onView(withId(R.id.topic_tabs_viewpager_container)).check(matches(isDisplayed()))
+    }
+  }
+
+  @Test
+  fun testTopicInfoFragment_loadFragment_configurationLandscape_imageViewNotDisplayed() {
+    launchTopicActivityIntent(TEST_TOPIC_ID).use {
+      onView(ViewMatchers.isRoot()).perform(orientationLandscape())
```

Comment:
For better code clarity write it like this:
`onView(isRoot()).perform(orientationLandscape())`

Do this in other test cases too and accordingly remove unused imports.

Reply:
> For better code clarity write it like this:
> `onView(isRoot()).perform(orientationLandscape())`
> 
> Do this in other test cases too and accordingly remove unused imports.

@rt4914 Changes done. Do merge it and assign me a new issue. Thank you :)